docs/content/migration.md

```diff
@@ -0,0 +1,545 @@
+
+The package contains the following:
+- A [Declarative API][116] that includes attributes like `fxLayout`, `fxFlex`, `fxHide`, `fxShow`,
+  and more.
+- A [Responsive API][117] that includes attributes like `fxLayout.lt-md`, `fxFlex.gt-sm`,
+  `fxHide.gt-xs`, `[ngClass.sm]`, `[ngStyle.xs]`, and more.
+- Both APIs include [CSS Grid][65] features in addition to the [CSS Flexbox][flexbox] features from
+  AngularJS Material.
+- The ability to define [Custom Breakpoints][118]
+- A [MediaObserver service][119] that allows subscribing to `MediaQuery` activation changes
+
+Since AngularJS Material's inception, browser support for [CSS Flexbox][flexbox] has [broadly
+improved][177] and [CSS Grid][65] has become [widely available][178]. Application developers should
+carefully consider the the costs of additional run-time dependencies versus native alternatives
+when choosing an approach to layout. This library adds a nontrivial payload cost.
+
+#### <a name="cdk-layout"></a> Angular CDK Layout
+The CDK's layout capabilities offer a lightweight alternative (or complement) to larger,
+full-blown layout systems.
+
+The CDK's [Layout][121] capability offers:
+- A `MediaMatcher` service that provides access to the low level, native [`MediaQueryList`][123]
+- A set of `Breakpoints` that match the [Material Design][122] responsive layout grid breakpoints
+- A `BreakpointObserver` service that offers APIs for responding to changes in viewport size based
+  on predefined breakpoints.
+
+### <a name="typography"></a> Typography
+
+AngularJS Material [typography classes][107] align with the Angular Material
+[typography classes][27].
+
+Angular Material also adds the following typography features:
+- [Customization via Sass mixins][174] that are similar to the theme configuration mixins
+- Utility Sass mixins and functions for use in [styling your custom components][175]
+  
+## <a name="comparison"></a> Comparison of Features and Migration Tips
+
+Angular Material introduces a number of [new components](#new-components) and features that do not
+exist in AngularJS Material. Angular Material also makes greater use of semantic HTML and native
+elements to improve accessibility.
+
+### Directives
+Most components use the same name between the two libraries, changing the `md-` prefix to `mat-`.
+For example, `<md-card>` becomes `<mat-card>`. Some element directives changed to attribute
+directives, for instance `<md-subheader>` changed to `matSubheader`.
+
+#### Component comparison reference 
+
+| Directive          | Old (`md-`)  | New (`mat-`) | Special Notes                                          |
+|--------------------|--------------|--------------|--------------------------------------------------------|
+| autocomplete       |   [Docs][41] |   [Docs][24] |                                                        |
+| autofocus          |   [Docs][69] |   -          | Not implemented                                        |
```

Comment:
cdkFocusInitial (Not a directive but works in numerous scenarios such as dialogs for specifying the initially focused element)

Reply:
Ah yeah, https://material.angular.io/cdk/a11y/overview#regions seems like it would work. I don't think that I saw it because it [wasn't in the API docs](https://github.com/angular/material2/issues/15799).